File: SecondYear/sauban123_MohdSauban_2226csit1130_2/Week_2/Fibonacci.cpp
vector<int> generateFibonacciNumbers(int n) {
    // Write your code here.
    vector<int> ans(n,0); //create a vector of size n and init.with "0"
    for(int i=1;i<n;i++){
        if(i==1){
            ans[i] =1;
        }
        else{
            ans[i] += ans[i-1] +ans[i-2];
        }
    }
    return ans;
}
//code ends here 

File: SecondYear/AvantikaSharma2307_AvantikaSharma_2226cseai1008_2/Week_2/fibonacci.py
from typing import List

def generateFibonacciNumbers(n: int) -> List[int]: 
    # Write your code here
    a=-1
    b=1
    i=1
    List=[]
    while(i<=n):
            c=a+b       
            a=b
            b=c
            List.append(c)
            i=i+1

    return List        